tools/clang/unittests/HLSL/CompilerTest.cpp

```diff
@@ -3599,10 +3598,6 @@ TEST_F(CompilerTest, CodeGenFloatingPointEnvironment) {
 
 #endif  //  defined(_WIN32) && !defined(_M_ARM64)
 
-TEST_F(CompilerTest, CodeGenInclude) {
-  CodeGenTestCheck(L"Include.hlsl");
-}
-
```

Comment:
Why was this test removed?

Reply:
This test was removed because it was actually checking stderr for the displayed include process text. The File Check tests are checking the disassembly and the remarks don't go there, so Adam suggested I just add a command test instead.

Reply:
We should probably add a new test in CompilerTest to check for the remark output.